### PR TITLE
Jon/ansible bucket

### PR DESCRIPTION
**Pull request by jon918**

### CODE REVIEW

File: examples/ssm-instance-access/main.tf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 0.12.7"
+}
+
+provider "aws" {
+  version = "~> 2.0"
+  region  = var.aws_region
```
Comment (yasyf) [inline examples/ssm-instance-access/main.tf:7]: can we default this to `us-east-1` to match ansible / most people's default cc @rickducott

Comment (rickducott) [reply examples/ssm-instance-access/main.tf:7]: https://linear.app/symops/issue/SYM-634/put-the-send-command-bucket-in-us-east-1

File: modules/ssm-instance-access/main.tf [hunk 2]
```diff
@@ -45,8 +45,34 @@ resource "aws_iam_policy" "ssm_instance_policy" {
               "ec2messages:SendReply"
           ],
           "Resource": "*"
+      },
+      {
+          "Effect": "Allow",
+          "Action": [
+              "s3:GetObject",
+              "s3:GetObjectAcl",
+              "s3:PutObject",
```
Comment (yasyf) [inline modules/ssm-instance-access/main.tf:54]: need `DeleteObject`

File: modules/ssm-user-access/main.tf [hunk 3]
```diff
@@ -74,9 +74,23 @@ data "aws_iam_policy_document" "ssm_user" {
...
+    ]
+    resources = [
+      "arn:aws:s3:::sym-doctor-*/*",
+    ]
+  }
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:GetObject",
+      "s3:GetObjectAcl",
```
Comment (yasyf) [inline modules/ssm-user-access/main.tf:87]: need `DeleteObject`

**Review by yasyf — CHANGES_REQUESTED** [review]

File: examples/ssm-instance-access/main.tf (hunk 1 above)
Comment (jon918) [reply examples/ssm-instance-access/main.tf:7]: we *could* but this is an example and we don't actually run this example as part of our stuff.

**Review by druiz — APPROVED** [review]

**Pull request merged**